app/waf/src/services/do.service.ts

```diff
@@ -325,6 +325,28 @@ export class OnboardingManager {
       }
       return target;
     },
+
+    users: (
+      target: TypeDOClassDeclaration['Common'],
+      obData: Adc,
+      additionalInfo?: object,
+    ): TypeDOClassDeclaration['Common'] => {
+      if (obData.compute.sshKey) {
+        let userInfo = {
+          class: 'User',
+          userType: 'root',
+          oldPassword: obData.management.connection!.rootPass,
+          newPassword: obData.management.connection!.rootPass,
+          keys: [obData.compute.sshKey!],
+        };
+
+        this.logger.debug('Add users part.');
+        return Object.assign(target, {root: userInfo});
+      } else {
+        this.logger.debug(`No provision found.`);
```

Comment:
Does it mean `No sshkey found.`?

Reply:
yes, I meant to log 'no provisioning of sshKey found'.

Reply:
why not changing it? 
From that statement 'No provision found.' in log, no one knows it refers to sshkey.